oeps/oep-0032-arch-unique-identifier-for-users.rst

```diff
@@ -0,0 +1,153 @@
+###################################
+OEP-32: Unique Identifier for Users
+###################################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-32 <oep-0032-arch-unique-identifier-for-users>`
+   * - Title
+     - Unique Identifier for Users
+   * - Last Modified
+     - 2019-01-28
+   * - Authors
+     - Brian Wilson, Robert Raposa, Nimisha Asthagiri
+   * - Arbiter
+     - Olga Stroilova
+   * - Status
+     - Draft
+   * - Type
+     - Architecture
+   * - Created
+     - 2019-01-25
+   * - `Review Period`
+     - 2019-01-28 - 2019-02-11
+
+Decision
+========
+
+We will use the Open edX user_id (a.k.a. *LMS user_id*) to uniquely identify a user in the Open edX system. The *LMS user_id* is the **id** value of the user's row in the `Django auth_user`_ table in the Open edX LMS.
+
+The *LMS user_id* should be used for all internal and public events, REST APIs, and JavaScript APIs.
+
+.. _Django auth_user: https://docs.djangoproject.com/en/2.0/topics/auth/default/#user-objects
+
+
+Context
+=======
+
+In 2016, the analytics team wrote a decision to use the *LMS user_id* in `Using consistent user identifiers in Segment events`_.
+
+In addition to needing a consistent user identifier for internal events, we also need one for public events, REST APIs and JavaScript APIs.
+
+.. _Using consistent user identifiers in Segment events: https://openedx.atlassian.net/wiki/spaces/AN/pages/144441849/Using+consistent+user+identifiers+in+Segment+events
```

Comment:
Is it possible to make this wiki page public?

Reply:
All necessary information exists in this OEP, so I killed the pointer to Confluence, since it pointed to other docs, and I didn't want to deal with all of that. :)